packages/graphql/src/translate/field-aggregations/apoc-run-utils.ts

```diff
@@ -1,5 +1,6 @@
 import { escapeQuery } from "./utils";
 import { AggregationAuth } from "./field-aggregations-auth";
+import { serializeObject } from "../utils";
 
 /** Wraps a query inside an apoc call, escaping strings and serializing params */
 export function wrapApocRun(query: string, params: Record<string, string> = {}): string {
```

Comment:
Slightly more detailed naming?

```suggestion
export function wrapWithApocRunFirstColumn(query: string, params: Record<string, string> = {}): string {
```